charts/opentelemetry-operator/templates/deployment.yaml

```diff
@@ -16,10 +16,14 @@ spec:
     metadata:
       annotations:
         kubectl.kubernetes.io/default-container: manager
+        {{- if .Values.manager.podAnnotations }}
         {{- include "opentelemetry-operator.podAnnotations" . | nindent 8 }}
+        {{- end }}
       labels:
         {{- include "opentelemetry-operator.selectorLabels" . | nindent 8 }}
+        {{- if .Values.manager.podLabels }}
```

Comment:
just to be able to evade `nindent` if properties are blank

Reply:
as alternative to conditions I could try to move `nindent` into the helper method(s) instead, let me know if I should do something

Reply:
this works